pkg/asset/agent/manifests/agentclusterinstall.go

```diff
@@ -323,6 +312,15 @@ func (a *AgentClusterInstall) finish() error {
 		return errors.Wrapf(err, "invalid PlatformType configured")
 	}
 
+	agentClusterInstallData, err := yaml.Marshal(agentClusterInstall)
```

Comment:
Will need to pass in `agentClusterInstall` (or add it to` AgentClusterInstall struct`).

Reply:
:man_facepalming: thanks.
It's already passed in, just as `a.Config`